docs/reference/errors-and-warnings/NU3035.md

```diff
@@ -0,0 +1,22 @@
+---
+title: NuGet Warning NU3035
+description: NU3035 Warning code
+author: heng-liu
+ms.author: henli
+ms.date: 3/18/2019
+ms.topic: reference
+ms.reviewer: dtivel
+f1_keywords: 
+  - NU3035
+---
+
+# NuGet Warning NU3035
+
+### Issue
+
+A complete certificate chain could not be built for the repository countersignature's signing certificate.
+
+
+### Solution
+
+Try to re-sign and timestamp the package. If the problem persists, contact the timestamp authority to discover the source of the problem.
```

Comment:
Just:

    Contact the repository that countersigned the package.

Reply:
Thanks! Fixed.